userspace/falco/configuration.cpp

```diff
@@ -176,6 +176,8 @@ void falco_configuration::init(string conf_filename, list<string> &cmdline_optio
 
```

Comment:
I had a chance to discuss feedback that got addressed by @leogr - this looks ready to me now 

LGTM